tarball_app_builder_xenial_v1/dxapp.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "tarball_app_builder_xenial_v1",
+  "title": "Tarball App Builder (Xenial v1)",
+  "summary": "Builds an app from a tarball",
+  "categories": ["Debugging"],
+  "dxapi": "1.0.0",
+  "billTo": "org-dnanexus",
```

```suggestion
  "billTo": "org-dnanexus_apps",
```

Please bill these new apps to `org-dnanexus_apps`, an org dedicated to apps+public app project billing. Thanks!

---

Is `org-dnanexus` still used for developers?

---

`org-dnanexus` should be removed from developers.